tests/security/apparmor/aa_autodep.pm

```diff
@@ -24,6 +24,8 @@ use version_utils qw(is_sle is_opensuse);
 sub run {
     my ($self) = @_;
 
+    zypper_call('in nscd');
```

Comment:
neatpicking, but for readability the call should be below the assignations that the method would use in different parts.

Reply:
what do you mean exactly? To move it to e.g. line 34 just above `assert_script_run`? 🤔

Reply:
after line 30.